descope/sdk/auth.go

```diff
@@ -199,6 +200,18 @@ type SAML interface {
 	ExchangeToken(ctx context.Context, code string, w http.ResponseWriter) (*descope.AuthenticationInfo, error)
 }
 
+type SSOSP interface {
```

Comment:
I'm thinking if better to call it `SSOServiceProvider` instead of `SSOSP`, since it might be easier to understand and read.

Reply:
sounds good. done